tex/paper.tex

```diff
@@ -120,10 +141,11 @@ \subsection{Review of the literature}\label{sec:review}
             Year     & Reference           & Number of Strategies & Type     & Source Code\\
             \midrule
             1979     & \cite{Axelrod1980a} & 13                   & Standard & Not immediately available\\
-            1979     & \cite{Axelrod1980b} & 64                   & Standard & Not immediately available\\
+            1979     & \cite{Axelrod1980b} & 64                   & Standard & Available in FORTRAN\\
             1991     & \cite{Bendor1991}   & 13                   & Noisy    & Not immediately available\\
             2002     & \cite{Stephens2002} & 16                   & Wildlife & Not applicable\\
-            % Is this worth including? If so I think we need more.
+            2005     & \cite{kendall2007iterated} & 223           & Varied   & Not available \\
+            2012     & \cite{Stewart2012} & 13                    & Standard & Not fully available \\
```

Comment:
Based on what you said in intro, should slany2007some be here? Haven't read the paper.